packs/abomination-vaults-bestiary/morlock-engineer.json

```diff
@@ -554,7 +554,7 @@
                 },
                 "category": "offensive",
                 "description": {
-                    "value": "<p>The morlock engineer tinkers with an adjacent construct or mechanical hazard. They attempt a @Check[crafting|defense:fortitude] check against the construct's or hazard's Fortitude DC. The morlock can't get an outcome better than failure if the target's level is more than double the morlock's.</p>\n<p>This ability reflects hasty battlefield repairs; once a construct or hazard regains Hit Points from this ability, it can't do so again until it's been @UUID[Compendium.pf2e.actionspf2e.Item.Repair]{Repaired}.</p>\n<hr />\n<p><strong>Critical Success</strong> The target regains [[/r 8d6[healing]]] Hit Points and a +1 circumstance bonus to attack rolls for 1 minute. Alternately, the morlock can deal [[/r 8d6]]{8d6 damage} (bludgeoning, piercing, or slashing, as chosen by the morlock engineer) to the construct or hazard.</p>\n<p><strong>Success</strong> As critical success, but the target regains [[/r 4d6[healing]]] Hit Points or the morlock deals [[/r 4d6]]{4d6 damage}.</p>\n<p><strong>Critical Failure</strong> The morlock injures themself, taking [[/r 3d6]]{3d6 damage} (typically bludgeoning, piercing, or slashing, but potentially a different type at the GM's discretion).</p>\n<p>@UUID[Compendium.pf2e.bestiary-effects.Item.Effect: Uncanny Tinker]</p>"
+                    "value": "<p>The morlock engineer tinkers with an adjacent construct or mechanical hazard. They attempt a @Check[crafting|defense:fortitude] check against the construct's or hazard's Fortitude DC. The morlock can't get an outcome better than failure if the target's level is more than double the morlock's.</p>\n<p>This ability reflects hasty battlefield repairs; once a construct or hazard regains Hit Points from this ability, it can't do so again until it's been @UUID[Compendium.pf2e.actionspf2e.Item.Repair]{Repaired}.</p>\n<hr />\n<p><strong>Critical Success</strong> The target regains @Damage[8d6[healing]] Hit Points and a +1 circumstance bonus to attack rolls for 1 minute. Alternately, the morlock can deal [[/r 8d6]]{8d6 damage} (bludgeoning, piercing, or slashing, as chosen by the morlock engineer) to the construct or hazard.</p>\n<p><strong>Success</strong> As critical success, but the target regains @Damage[4d6[healing]] Hit Points or the morlock deals [[/r 4d6]]{4d6 damage}.</p>\n<p><strong>Critical Failure</strong> The morlock injures themself, taking [[/r 3d6]]{3d6 damage} (typically bludgeoning, piercing, or slashing, but potentially a different type at the GM's discretion).</p>\n<p>@UUID[Compendium.pf2e.bestiary-effects.Item.Effect: Uncanny Tinker]</p>"
```

Comment:
```suggestion
                    "value": "<p>The morlock engineer tinkers with an adjacent construct or mechanical hazard. They attempt a @Check[crafting|defense:fortitude] check against the construct's or hazard's Fortitude DC. The morlock can't get an outcome better than failure if the target's level is more than double the morlock's.</p>\n<p>This ability reflects hasty battlefield repairs; once a construct or hazard regains Hit Points from this ability, it can't do so again until it's been @UUID[Compendium.pf2e.actionspf2e.Item.Repair]{Repaired}.</p>\n<hr />\n<p><strong>Critical Success</strong> The target regains @Damage[8d6[healing]] Hit Points and a +1 circumstance bonus to attack rolls for 1 minute. Alternately, the morlock can deal @Damage[8d6[untyped]] damage (bludgeoning, piercing, or slashing, as chosen by the morlock engineer) to the construct or hazard.</p>\n<p><strong>Success</strong> As critical success, but the target regains @Damage[4d6[healing]] Hit Points or the morlock deals @Damage[4d6[untyped]] damage.</p>\n<p><strong>Critical Failure</strong> The morlock injures themself, taking @Damage[3d6[untyped]] damage (typically bludgeoning, piercing, or slashing, but potentially a different type at the GM's discretion).</p>\n<p>@UUID[Compendium.pf2e.bestiary-effects.Item.Effect: Uncanny Tinker]</p>"
```